test/router.js

```diff
@@ -2,11 +2,18 @@
 var assert = require('assert');
 var _ = require('lodash');
 var sinon = require('sinon');
-var Router = require('../lib/router');
+var helpers = require('./helpers');
+var fakeReadPkgUp = helpers.fakeReadPkgUp();
+var proxyquire = require('proxyquire');
+var Router = proxyquire('../lib/router', {
+  'read-pkg-up': fakeReadPkgUp
```

Comment:
I'm not sure if I understand. Are you suggesting (1) to get ride of line 6 by inlining it in line 9 or (2) to not include the fake read-pkg-up module in helpers but inlining that instead?